react/features/base/config/actions.js

```diff
@@ -10,6 +10,24 @@ import { CONFIG_WILL_LOAD, LOAD_CONFIG_ERROR, SET_CONFIG } from './actionTypes';
 import { _CONFIG_STORE_PREFIX } from './constants';
 import { setConfigFromURLParams } from './functions';
 
+
+/**
+ * Updates the config with new options.
+ *
+ * @param {Object} options - The new options (to add).
+ * @returns {Function}
+ */
+export function amendConfig(options: Object) {
```

Comment:
Search for `_UPDATE_CONFIG`. We have this machinery already, albeit in a bit of a twisted way. If we want to make this a valid internal API we should consolidate it.

Reply:
Sorry, missed that.

I have made public the `updateConfig` action and used it.

Reply:
Don't be sorry, you couldn't have known!